### ISSUE TITLE
Dockerfile missing to install rsync (used in ovpn_copy_server_files)

### ISSUE THREAD

**Issue by benzht** [kylemanna/docker-openvpn#73]
The command ovpn_copy_server_files fails because rsync is not installed in the docker image.

**Comment by kylemanna** [comment]
This should be working now.  

@ypid can you make sure my changes work as expected.  Also, this `ovpn_copy_server_files` script seems like a subset of the backup script and has issues with permissions (i.e. creates files as root), is it really necessary?  Nobody has used it since rsync was never in the build.

Nevertheless, I've added a unit-test for it to prevent breakage in the future.

**Comment by ypid** [comment]
@kylemanna Thanks for keeping this project maintained :+1: including scripts which apparently only I use …
I can confirm that your changed version works as expected. About the missing rsync. I used the script myself outside of docker. Not every consistent but I saw now real benefit in doing it with docker as I wrote the script and know that it is doing and keep the CA not on the server itself.

The script is not redundant. See: https://github.com/kylemanna/docker-openvpn/blob/master/docs/paranoid.md#keep-the-ca-root-key-save

Thanks.
